packages/amplication-client/src/VersionControl/PendingChangeContent.tsx

```diff
@@ -23,7 +23,7 @@ const PendingChangeContent = ({
 }: Props) => {
   const { currentWorkspace, currentProject } = useContext(AppContext);
 
-  const url = `/${currentWorkspace?.id}/${currentProject?.id}/${change.resource.id}/${relativeUrl}`;
+  const url = `/${currentWorkspace?.id}/${currentProject?.id}/settings/general`;
```

Comment:
@HeetVekariya I tested it locally, and I realized that we actually can't hardcode it because we have more than one type of "change". There are changes like entity, plugin, service settings, and there is a type of services named `ProjectConfiguration`. As a result, this change will fix the project settings link, but the other links in the pending changes area will refer to this link, which is not true.

The project settings link should redirect to a resource type `ProjectConfiguration`. Therefore, what you need to do is to distinguish between this type and the "other" types.

For now, I think the easiest solution is:

```suggestion
  const isProjectConfigResourceType =
    change.resource.resourceType ===
    models.EnumResourceType.ProjectConfiguration;

  const url = isProjectConfigResourceType
    ? `/${currentWorkspace?.id}/${currentProject?.id}/settings/general`
    : `/${currentWorkspace?.id}/${currentProject?.id}/${change.resource.id}/${relativeUrl}`;
```

Reply:
- I have made the suggested changes, but  will this ```/${currentWorkspace?.id}/${currentProject?.id}/${change.resource.id}/${relativeUrl}``` close issue #7254 
- Should it not be this: ```/${currentWorkspace?.id}/${currentProject?.id}/${relativeUrl}```